omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -64,6 +64,14 @@
     }
     <% end -%>
 
+    <% if node['private_chef']['required_recipe']['enable'] -%>
+    location ~ /organizations/([^/]+)/required_recipe {
```

Comment:
I can't imagine how this would ever become an issue, but since we all _think_ that this location regex starts at the beginning of the request path, can you add a `^` there?